app/lib/dartdoc/dartdoc_runner.dart

```diff
@@ -244,7 +243,7 @@ class DartdocJobProcessor extends JobProcessor {
       await toolEnvRef.release();
     }
 
-    final coverage = calculateCoverage(dartdocData);
+    final coverage = dartdocData.coverage ?? Coverage(total: 0, documented: 0);
```

Comment:
When can `dartdocData.coverage` be `null`? When loading old data?

Can we bump the runtime version or somehow avoid this? I'm not a fan of sprinkling obscure conditions all over the code.

Reply:
Good point, here this can't be `null`. But reading extracted data from old reports may make it null. I'll update the comment.